docs/estimators/index.rst

```diff
@@ -64,13 +65,17 @@ Quantity          Definition
 norm_errp		  Positive error of the norm
 norm_errn	      Negative error of the norm
 norm_ul			  Upper limit of the norm
+stat			  Fit statistics value of the best fit hypothesis
+stat_null		  Fit statistics value of the null hypothesis
+================= =================================================
+npred		  	  Predicted counts of the best fit hypothesis
+npred_null        Predicted counts of the null hypothesis
+npred_signal      Predicted counts of the signal over `npred_bull`, equivalent to (`npred - npred_null`)
```

Comment:
I think `signal` is misleading. Because `npred_null` can contain signal (in the sense of astrophysical photons). I still think `excess` is better, since it is really what it is.

Reply:
Yes, I agree. My motivation was to stay in-line with what we define here: https://docs.gammapy.org/0.18.2/stats/index.html#notations but I now realise this requires some more thought. I just checked and `npred` is also define here https://gamma-astro-data-formats.readthedocs.io/en/latest/spectra/flux_points/index.html#normalization-representation, where it means the predicted counts from the model component only (so `npred - npred_null` in our definition...).

It's a bit similar with excess, because `MapDataset.excess` is already defined as `counts - background` and does not account for any additional source contributions. So that definition of `excess` is somewhat variable, in the sense that it can either include additional source contributions or not. I wonder whether we can find a fully consistent definition. I'll think about in once more and make a new proposal...

Reply:
I think we could rename `sig` -> `excess` consistently here: https://docs.gammapy.org/0.18.2/stats/index.html#notations, to get rid of the ambiguity with n sigma. But this would require adapting the API of `MapDataset` and `CountsStatistic` once again (fine from my side...if not now, we will not do it for v1.0...)

Reply:
I think the inconsistency between "our" `npred` (which means the total predicted counts...) and the definition in gadf cannot easily be resolved, it would require to either re-define in Gammapy, which I don't want to do, because `MapDataset.npred()` is already and establish part of our API (`.npred_signal()` possibly not that much...) or in gadf.

Reply:
What about if we just stay in line with the `MapDataset` and use `npred`, `npred_background` and `npred_excess`, but clearly state, that for estimators by default a "residual convention" like `npred_background = dataset.npred()` is used? In case there are no source models defined it is equivalent anyway. When looking at the map users will see whether sources are included in the background anyway...

Reply:
On 2nd thought maybe `npred`, `npred_excess` and `npred_null` is also fine...

Reply:
I renamed to `npred_excess`...